docs/v1/P4Runtime-Spec.mdk

```diff
@@ -3610,10 +3618,11 @@ An `ActionProfileGroup` entity update message has the following fields:
       existing port on the device, otherwise the server must return
       `INVALID_ARGUMENT`.
 
-* `max_size` is the maximum sum of all member weights for the group. This field
-  is defined when the group is inserted, and must not be changed in a
-  `MODIFY` update. See the subsection below for the [rules on setting
-  `max_size`](#sec-max-size-rules).
+* `max_size` is the maximum sum of all members or member weights (as per the
```

Comment:
I guess this is not specific to this PR, but we forgot to make that change in the previous PR?

Reply:
Yes, that's right. The INVALID_ARGUMENT_ERROR part was implied below, but I got some internal comments of confusion so I thought I'd clarify at the same time.